src/main/java/org/jenkinsci/plugins/mesos/MesosCloud.java

```diff
@@ -64,6 +57,7 @@
 
 public class MesosCloud extends Cloud {
   private static final String DEFAULT_DECLINE_OFFER_DURATION = "600000"; // 10 mins.
+  public static final double SHORT_DECLINE_OFFER_DURATION_SEC = 5;
```

Comment:
Should this be exposed as a config option like `DEFAULT_DECLINE_OFFER_DURATION`?

Reply:
Possibly.  It is the default decline behavior of Mesos which is already used implicitly elsewhere in this framework without being configurable.

I think it's a nice to have that can follow on after this PR.